packages/shared/src/components/sidebar/sections/MainSection.tsx

```diff
@@ -42,18 +46,19 @@ export const MainSection = ({
         }
       : undefined;
 
-    const plus = isEnrolledNotPlus
-      ? {
-          title: 'Upgrade to Plus',
-          path: `${webappUrl}plus`,
-          onClick: onPlusClick,
-          isForcedLink: true,
-          requiresLogin: true,
-          icon: <DevPlusIcon />,
-          color:
-            'text-action-plus-default bg-action-plus-float hover:bg-action-plus-hover active:bg-action-plus-active',
-        }
-      : undefined;
+    const plus =
+      isEnrolledNotPlus && !isPlusEntrypointExperiment
+        ? {
+            title: 'Upgrade to Plus',
+            path: `${webappUrl}plus`,
+            onClick: onPlusClick,
+            isForcedLink: true,
+            requiresLogin: true,
+            icon: <DevPlusIcon />,
+            color:
```

Comment:
Maybe change to className as we passing many classNames.

Reply:
This line is unchanged from before, I can rename this property but this will require a bit of work refactoring `SidebarMenuItem`. If is ok I can go ahead ✔️

Reply:
This is highlighted because I've add a `&& !isPlusEntrypointExperiment` condition

Reply:
Ah ok, no let's leave then :)